x-pack/legacy/plugins/actions/server/mappings.json

```diff
@@ -2,7 +2,12 @@
   "action": {
     "properties": {
       "name": {
-        "type": "text"
+        "type": "text",
```

Comment:
These mapping changes won't require any kind of migration, since they're additive, right?

Reply:
That's correct, when `.kibana` migrates from ex: `_1` to `_2`, the data will be analyzed differently but no migration necessary.